### PR TITLE
A little escalator update.

### PR DESCRIPTION
**Pull request by NatureGecko**
Changes made.
- New model for the escalator. 
- Stop and change direction of the escalator using a brush. ( forward > stop > backward instead of only forward and backward )
- Change escalator block render option from "solid" to "cutoff".

https://user-images.githubusercontent.com/59920578/227006038-39c315b8-f1be-4ed2-a3ad-a6e210c2ee74.mp4

### CODE REVIEW

**Comment by Ultimate-USA-YT** [comment]
This is Impressive!

File: common/src/main/java/mtr/MTRClient.java
Comment (EpicPuppy613) [inline common/src/main/java/mtr/MTRClient.java:1]: Did you intend to put the new escalator cutout renderer under the not lifts only section?

**Review by EpicPuppy613 — CHANGES_REQUESTED** [review]
`registerBlockRenderType` for `Blocks.ESCALATOR_STEP` and `ESCALATOR_SIDE` should be moved outside of the `!Keys.LIFTS_ONLY` section.

**Comment by NatureGecko** [comment]
> [EpicPuppy613](/EpicPuppy613)
 :)
Got the point and moved to their proper place, thanks for your suggestion.

**Review by EpicPuppy613 — APPROVED** [review]
Much better, keep up the good work!

**Pull request merged**